cli/actions/lint.js

```diff
@@ -1,42 +1,29 @@
 
 // Command to lint src code
 
-const CLIEngine = require('eslint').CLIEngine;
+const shell = require('shelljs');
 const path = require('path');
 const logger = require('./../logger');
 const glob = require('glob');
 const { userRootPath } = require('../../utils/paths')();
 
-module.exports = () => {
+module.exports = (config, flags) => {
   const eslintrc = glob.sync(`${userRootPath}/.*eslintrc*`);
   const configFile = eslintrc.length
       ? eslintrc[0]
       : path.join(__dirname, '../../config/.eslintrc.base.json');
 
   logger.info(`Using ESLint file: ${configFile}`);
 
-  // http://eslint.org/docs/developer-guide/nodejs-api
-  const eslintCLI = {
-    envs: ['browser'],
-    extensions: ['.js'],
-    useEslintrc: true,
-    configFile,
-  };
-
-  // Get the default dir or the dir specified by the user/-d.
   const lint = () => {
-    const files = ['src/'];
-    const cli = new CLIEngine(eslintCLI);
-    const report = cli.executeOnFiles(files);
-    const formatter = cli.getFormatter();
-    logger.log(`${formatter(report.results)}\n`);
-
-    if (report.errorCount === 0) {
-      logger.end(`Your JS looks ${report.warningCount === 0 ? 'great ✨' :
+    const cmd = `eslint src/ -c ${configFile} --color --env browser ${flags.join(' ')}`;
```

Comment:
I think it might be better to put `env: { browser: true}` in the base eslint config instead, this way it's overridable at the command-line by the user.

I believe color is enabled by default anyway?